Libraries/CustomComponents/ListView/ListView.js

```diff
@@ -577,7 +577,10 @@ var ListView = React.createClass({
     }
     if (updatedFrames) {
       updatedFrames.forEach((newFrame) => {
-        this._childFrames[newFrame.index] = merge(newFrame);
+        // Sometimes x/y/width/heigh maybe return zero
+        if (newFrame.x != 0 || newFrame.y != 0 || newFrame.width != 0 || newFrame.height != 0) {
```

Comment:
If coordinate as 0 and size is 0 too,it will be excluded.
The reason for this change is sometimes some frame‘s data maybe lost or frame's size & coordinate is overridden by 0.
So it's possible, the frame's data width & height = 0 for coordinate as 0,but it is an error data.